src/styles/utils/normalize.scss

```diff
@@ -0,0 +1,10 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: $cardsFontStyle;
+}
```

Comment:
only `box-sizing: border-box;` is allowed to use at `*` selector

Reply:
how can i reset margin for all el?

Reply:
You can reset margins for elements you are using, for example:

```
ul {
  margin: 0;
}
```